docs/tutorial/correlatedstacks.rst

```diff
@@ -8,21 +8,40 @@ Correlated stacks
 Bluelake has the ability to export videos from the camera's.
 These videos can be opened and sliced using `CorrelatedStack`::
 
-    stack = lk.CorrelatedStack("example.tiff")  # Loading a stack.
+    stack = lk.CorrelatedStack("cas9_wf.tiff")  # Loading a stack.
     stack_slice = stack[2:10]  # Grab frame 2 to 9
 
+You can quickly plot an individual frame using the `plot()` method::
+
+    stack.plot(frame=0, channel="rgb")
```

Comment:
Oh, I just realized that we have an inconsistency between the scan and correlated stack APIs. The scans number from 1 and the stacks from 0 :( I see that the plot also indicates that this is frame 1 even though the parameter is given as `frame=0`. How big of a break would it be to fix this inconsistency? Not for this PR in any case, I just noticed it here.